x-pack/plugins/remote_clusters/server/routes/api/update_route.ts

```diff
@@ -9,16 +9,21 @@ import { schema, TypeOf } from '@kbn/config-schema';
 import { i18n } from '@kbn/i18n';
 import { RequestHandler } from 'src/core/server';
 
-import { API_BASE_PATH } from '../../../common/constants';
-import { serializeCluster, deserializeCluster } from '../../../common/lib';
+import { API_BASE_PATH, SNIFF_MODE, PROXY_MODE } from '../../../common/constants';
+import { serializeCluster, deserializeCluster, Cluster, ClusterEs } from '../../../common/lib';
 import { doesClusterExist } from '../../lib/does_cluster_exist';
 import { RouteDependencies } from '../../types';
 import { licensePreRoutingFactory } from '../../lib/license_pre_routing_factory';
 import { isEsError } from '../../lib/is_es_error';
 
 const bodyValidation = schema.object({
-  seeds: schema.arrayOf(schema.string()),
   skipUnavailable: schema.boolean(),
+  mode: schema.oneOf([schema.literal(PROXY_MODE), schema.literal(SNIFF_MODE)]),
+  seeds: schema.nullable(schema.arrayOf(schema.string())),
+  nodeConnections: schema.nullable(schema.number()),
```

Comment:
Did we miss this in the original NP migration? I'm going to make a mental note to pay extra attention to this when I'm doing my own migration work because it seems super-easy to overlook a missed validation during testing. It seems the only way for us to discover a missed validation is with thorough functional or manual testing, right?

Reply:
I actually added support for defining node connections as part of this PR. It was always possible via the ES API, but we were previously only allowing users to define `seeds` in the UI.

Reply:
Thanks, I missed that!